csFastFloat/FastFloatParser.cs

```diff
@@ -79,7 +79,7 @@ unsafe static public float ParseFloat(char* first, char* last, chars_format expe
 
     unsafe static internal float ParseNumber(char* first, char* last, chars_format expectedFormat = chars_format.is_general, char decimal_separator = '.')
     {
-      while ((first != last) && Utils.is_space((byte)(*first)))
+      while ((first != last) && Utils.is_ascii_space(*first))
```

Comment:
Once you change the signature to consider chars, you're in for a can of worms as there are numerous more white space characters outside the ascii space. At that rate is it better to be safe even if its slower and just use `char.IsWhiteSpace`?

Reply:
No. Because only support the default (C) locale.

The can of worms comes only if you want to support other locales.